js/pathfinding.js

```diff
@@ -237,7 +237,8 @@ export function isPathfindingEnabled() {
 
 export function findPath(from, to, token, previousWaypoints) {
 	if (canvas.grid.type === CONST.GRID_TYPES.GRIDLESS) {
-		let tokenSize = Math.max(token.data.width, token.data.height) * canvas.dimensions.size;
+		let radiusMultiplier = game.settings.get(settingsKey, "pathfindingRadius");
+		let tokenSize = Math.max(token.data.width, token.data.height) * canvas.dimensions.size * radiusMultiplier;
```

Comment:
I'd prefer having this multiplication (and the settings lookup) within the if-conditional below, so that only the token size that's passed into the initialization gets modified. This way we can retain the key of the map as an integer (which I'm more comfortable with that having a float as the map key).